FDD_RF_Module/FDD_RF_Modeling.py

```diff
@@ -47,17 +45,28 @@ class FDD_RF_Modeling():
             training and testing
 
     """
-    def __init__(self, weather = 'TN_Knoxville', labeling_methodology = 'Simple',
-     feature_selection_methodology = 'None', aggregate_n_runs = 4,
+    def __init__(self, configs, weather = 'TN_Knoxville', labeling_methodology = 'Simple',
+     feature_selection_methodology = 'None', fdd_reporting_frequency_hrs = 4,
      number_of_trees = 20, randomseed=2021):
+        self.configs = configs
         self.weather = weather
         self.labeling_methodology = labeling_methodology
         self.feature_selection_methodology = feature_selection_methodology
         self.number_of_trees = number_of_trees
-        self.aggregate_n_runs = aggregate_n_runs
+        self.fdd_reporting_frequency_hrs = fdd_reporting_frequency_hrs
         self.randomseed = randomseed
         self.root_path = os.getcwd()
 
+    def get_timeinterval(self, os_timestamp):
+    
+        #converting timestamp to pandas datetime
+        timestamp = pd.to_datetime(os_timestamp)
+        # inferring timestep (frequency) from the dataframe
+        dt = timestamp.diff().value_counts().idxmax() # in pandas timedelta
+        dt = int(dt.value/(10**9)/60) # in minutes
```

Comment:
What does (10**9) stand for?

Reply:
so data type of dt there is pandas timedelta. i just remembered I can use like .day attribute to convert the raw number of dt to "days" but the original unit of the dt is in "nano" seconds. so it's basically converting unit.